openstack/keystone/templates/etc/_secrets.conf.tpl

```diff
@@ -20,14 +20,14 @@ connection = {{ include "db_url_mysql" . }}
 [cache]
 memcache_sasl_enabled = True
 memcache_username = {{ .Values.memcached.auth.username }}
-memcache_password = {{ .Values.memcached.auth.password }}
+memcache_password = {{ .Values.memcached.auth.password | include "resolve_secret" }}
 {{- end }}
 
 {{- if not (and (hasKey $.Values "oslo_messaging_notifications") ($.Values.oslo_messaging_notifications.disabled)) }}
 [oslo_messaging_notifications]
 driver = messaging
   {{- if and (.Values.audit.central_service.user) (.Values.audit.central_service.password) }}
-transport_url = rabbit://{{ .Values.audit.central_service.user }}:{{ .Values.audit.central_service.password }}@{{ .Values.audit.central_service.host }}:{{ .Values.audit.central_service.port }}/
+transport_url = rabbit://{{ .Values.audit.central_service.user }}:{{ .Values.audit.central_service.password | include "resolve_secret_urlquery" }}@{{ .Values.audit.central_service.host }}:{{ .Values.audit.central_service.port }}/
```

Comment:
`.Values.audit.central_service.user` here should be with `resolve_secret_urlquery`, because in some places it's taken from the vault.